project_liberation/main_page/static/main_page/main_screen.sass

```diff
@@ -0,0 +1,46 @@
+.jumbo
+	background-image: url('/static/main_page/images/main_screen/slide_1.jpg')
```

Comment:
Hmm - this is interesting case - how to handle static image which is declared in sass. I think that for this moment we can leave it like this but if you have any idea how to give static path to images by template tag fell free to use it :D

Reply:
@kbeker Probably, I could use style in-line in haml :P